static/app/views/alerts/rules/metric/actions.tsx

```diff
@@ -22,9 +22,9 @@ export function addOrUpdateRule(
   query?: object | any
 ) {
   const isExisting = isSavedRule(rule);
-  const endpoint = `/projects/${orgId}/${projectId}/alert-rules/${
-    isSavedRule(rule) ? `${rule.id}/` : ''
-  }`;
+  const endpoint = isExisting
+    ? `/projects/${orgId}/${projectId}/alert-rules/${rule.id}/`
+    : `/organizations/${orgId}/alert-rules/`;
```

Comment:
If I understand correctly, if the rule is not already existing, it will change the endpoint to `/organizations/${orgId}/alert-rules/`. Will `${projectId}` still be needed in the new endpoint? @ceorourke

Reply:
It's still passed in the request, it's just not in the URL :)